docs/user_guide/discretisation/IncreasingWidthDiscretiser.rst

```diff
@@ -0,0 +1,134 @@
+.. _increasing_width_discretiser:
+
+.. currentmodule:: feature_engine.discretisation
+
+IncreasingWidthDiscretiser
+==========================
+
+The :class:`IncreasingWidthDiscretiser()` divides continuous numerical variables into
+intervals of increasing width with equal increments. Note that the
+proportion of observations per interval may vary.
+
+This discretisation technique is great when the distribution of the variable is right skewed.
+
+The :class:`IncreasingWidthDiscretiser()` works only with numerical variables. A list of
+variables to discretise can be indicated, or the discretiser will automatically select
+all numerical variables in the train set.
+
+**Note**
+
+This discretiser could be numerically unstable when the values of the variable are small (their order of magnitude is small).
+
+**Example**
+
+Let's look at an example using the House Prices Dataset (more details about the
+dataset :ref:`here <datasets>`).
+
+Let's load the house prices dataset and  separate it into train and test sets:
+
+.. code:: python
+
+	import numpy as np
+	import pandas as pd
+	import matplotlib.pyplot as plt
+	from sklearn.model_selection import train_test_split
+
+	from feature_engine.discretisation import IncreasingWidthDiscretiser
+
+	# Load dataset
+	data = pd.read_csv('houseprice.csv')
+
+	# Separate into train and test sets
+	X_train, X_test, y_train, y_test =  train_test_split(
+		    data.drop(['Id', 'SalePrice'], axis=1),
+		    data['SalePrice'], test_size=0.3, random_state=0)
+
+
+Now we want to discretise the 2 variables indicated below into 10 intervals of equal
+width:
+
+.. code:: python
+
+	# set up the discretisation transformer
+	disc = IncreasingWidthDiscretiser(bins=10, variables=['LotArea', 'GrLivArea'])
+
+	# fit the transformer
+	disc.fit(X_train)
+
+With `fit()` the transformer learns the boundaries of each interval. Then, we can go
+ahead and sort the values into the intervals:
+
+.. code:: python
+
+	# transform the data
+	train_t= disc.transform(X_train)
+	test_t= disc.transform(X_test)
+
+The `binner_dict_` stores the interval limits identified for each variable.
+
+.. code:: python
+
+	disc.binner_dict_
+
+.. code:: python
+
+	'LotArea': [-inf,
+        1303.412,
+        1311.643,
+        1339.727,
+        1435.557,
+        1762.542,
+        2878.27,
+        6685.32,
+        19675.608,
+        64000.633,
+        inf],
+	 'GrLivArea': [-inf,
+        336.311,
+        339.34,
+        346.34,
+        362.515,
+        399.894,
+        486.27,
+        685.871,
+        1147.115,
+        2212.974,
+        inf]}
+
+With increasing width discretisation, each bin does not necessarily contain the same number of observations.
+It will also "skew" distribution. It could be useful to apply it to right skewed distribution
```

Comment:
when would it skew the distribution? Could we add a bit more detail?

Reply:
The graph below clearly shows how it works